elastic/kibana
# [ResponseOps][Accessibility] Ensures every ID attribute value is unique

Resolves https://github.com/elastic/kibana/issues/144953
## Summary

Changed the labels for pagerduty, x-matters, and teams logo svgs.

### Checklist


### To verify

- Log into the Kibana Stack Management > Rules page
- Click the Create New Rule button
- In the Create rule flyout, select a rule and make sure the list of connectors is showing
- Run the [axe browser plugin](https://deque.com/axe) (from Dev tools) when the vendor icons are on screen
- Verify that you see 0 duplicated ID errors instead of two.
